mac

```diff
@@ -54,12 +54,14 @@ exit_message() {
 trap exit_message EXIT
 
 ## Check OS X >= 10.10
-
-if [[ $(sw_vers -productVersion) != 10.1[0-1].* ]]; then
-  print_error "Unsupported OS X version. Please use OS X Yosemite or El Capitan."
-  exit 1
+if ! [ -n "$DANGER_ZONE" ]; then
```

Comment:
@sklise: don't need the `!` to negate the conditional.

`-n` means "has nonzero length". So `-n "$DANGER_ZONE"` basically means the variable *is set*. (`-z` is the opposite, meaning the variable has zero-length; or is unset.)